src/osgPlugins/OpenFlight/ReaderWriterFLT.cpp

```diff
@@ -1,40 +1,44 @@
-/* -*-c++-*- OpenSceneGraph - Copyright (C) 1998-2006 Robert Osfield
+/* -*-c++-*- OpenSceneGraph - Copyright (C) 1998-2006 Robert Osfield 
  *
- * This library is open source and may be redistributed and/or modified under
- * the terms of the OpenSceneGraph Public License (OSGPL) version 0.0 or
+ * This library is open source and may be redistributed and/or modified under  
+ * the terms of the OpenSceneGraph Public License (OSGPL) version 0.0 or 
  * (at your option) any later version.  The full license is in LICENSE file
  * included with this distribution, and on the openscenegraph.org website.
- *
+ * 
  * This library is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the 
  * OpenSceneGraph Public License for more details.
 */
 
 //
-// OpenFlight� loader for OpenSceneGraph
+// OpenFlight� loader for OpenSceneGraph
 //
 //  Copyright (C) 2005-2007  Brede Johansen
 //
 
 #include <stdexcept>
 #include <osg/Notify>
 #include <osg/ProxyNode>
+#include <osg/ConcurrencyViewerMacros>
 #include <osgDB/FileNameUtils>
 #include <osgDB/FileUtils>
 #include <osgDB/Registry>
 #include <osgDB/ReadFile>
 #include <OpenThreads/ReentrantMutex>
 #include <osgUtil/Optimizer>
+#include <vector>
 
 #include "Registry.h"
 #include "Document.h"
 #include "RecordInputStream.h"
 #include "DataOutputStream.h"
 #include "FltExportVisitor.h"
 #include "ExportOptions.h"
+#include "DtCDBSigSize.h"
+#include "DtCDBSigSizeTable.h"
 
-#define SERIALIZER() OpenThreads::ScopedLock<OpenThreads::ReentrantMutex> lock(_serializerMutex)
+#define SERIALIZER() OpenThreads::ScopedLock<OpenThreads::ReentrantMutex> lock(_serializerMutex)  
```

Comment:
We should take the 3.6 version of this change if we can.

Reply:
WIll take this change excepting anything in the following comments for this file.

Reply:
Pending.